.github/workflows/playbook-test.yml

```
@@ -24,7 +24,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
```

@dependabot likes the new v3 version:

```suggestion
      uses: actions/setup-python@v3
```